src/number_theory/number_field/embeddings.lean

```diff
@@ -123,3 +124,170 @@ end
 end bounded
 
 end number_field.embeddings
+
+namespace number_field
+
+section place
+
+variables {A : Type*} [normed_division_ring A] {K : Type*} [field K] (φ : K →+* A)
+
+/-- An embedding into a normed division ring defines a place of `K` -/
+def place : K → ℝ := norm ∘ φ
+
+lemma places.nonneg (x : K) : 0 ≤ place φ x := by simp only [place, norm_nonneg]
```

Comment:
I think you can call all of these lemmas `place.nonneg` and so on (instead of `placeS`). Also, yu can just use a `namespace` to save some characters (don't forget to move the definition *before* the namespace).